pkg/stanza/operator/parser/container/parser.go

```diff
@@ -265,6 +247,35 @@ func (p *Parser) handleAttributeMappings(e *entry.Entry) error {
 	return nil
 }
 
+func (p *Parser) cbGoTimeLayout(e *entry.Entry) error {
+	timeLayout := goTimeLayout
+	err := parseTime(e, timeLayout)
```

Comment:
Maybe this part can be moved to a separate method since it's exactly the same in `cbCrioTimeLayout`?

Reply:
sorry, did not quite understand. The `timeLayout` has different values in both functions right

Reply:
yeap, keep `timeLayout` separate and move the common part:

```go
func (p *Parser) cbGoTimeLayout(e *entry.Entry) error {
	return p.parseCallback(e, goTimeLayout)
}

func (p *Parser) cbCRITimeLayout(e *entry.Entry) error {
	return p.parseCallback(e, criTimeLayout)
}

func (p *Parser) parseCallback(e *entry.Entry, timeLayout string) error {
	err := parseTime(e, timeLayout)
	if err != nil {
		return fmt.Errorf("failed to parse time: %w", err)
	}

	err = p.handleAttributeMappings(e)
	if err != nil {
		return fmt.Errorf("failed to handle attribute mappings: %w", err)
	}
	return nil
}
```